Added Notifications for Machine Cycles

KurtSanders opened the pull request:
Added the ability for the user to select one or more machine cycles to get notifications.  Eliminates the need for users of this application to create Rules for basic machine cycle notifications.  Also added a new currentStatus_ variable for titlecase of the machine cycle.

KurtSanders committed:
Update ThinQ_Integration.groovy

Fixed syntax error

KurtSanders committed:
Update ThinQ_Fridge.groovy

Updated to show new fields for Dashboard that have the temp value and units

KurtSanders committed:
currentStateName capitalized()

Changed currentStateName to be capitalized for niceer display in Dashboards

KurtSanders committed:
Update ThinQ_Fridge.groovy

Fixed error with incorrent attribute definitions

KurtSanders committed:
Update ThinQ_Fridge.groovy

dcmeglio committed:
Removed debug logging that is no longer needed

dcmeglio committed:
Added attributes for fridge temp that are in display format.

KurtSanders committed:
Added Machine Cycle Notifications

KurtSanders committed:
Update ThinQ_Fridge.groovy

KurtSanders committed:
Update ThinQ_Fridge.groovy

KurtSanders committed:
Added User Selectable Machine Cycle Notifications

KurtSanders committed:
Update ThinQ_Integration.groovy

Added "end" to cycles

KurtSanders commented:
I'm confused, did you close this pull request because you are not interested in these new features?  Just curious?

dcmeglio commented:
No. I must have clicked the wrong button when merging in something else. I’m not home right now but I’ll fix it later. 

> On Feb 4, 2021, at 6:59 PM, Kurt Sanders <notifications@github.com> wrote:
> 
> ﻿
> I'm confused, did you close this pull request because you are not interested in these new features? Just curious?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or unsubscribe.

KurtSanders commented:
No rush, perhaps I can rebase against the new 'master' branch and resubmit...  

Your recent updates are awesome, thanks for developing this app...  My new LG dryer and LG washer are working perfectly for notifications and dashboard status.  

I decided to not integrate the V1 LG refrigerator since it really does not add any status values for my needs....  LG's V1 interface is archaic!   


¸¸.•*¨*•*´¨)
¸.•´¸.•*´¨) ¸.•*¨)
(¸.•´ (¸.•` ¤ 

Kurt Sanders
2485 Ridge Road
Xenia, Ohio 45385
+1937-723-0010

> On Feb 4, 2021, at 7:13 PM, dcmeglio <notifications@github.com> wrote:
> 
> 
> 
> 
> No. I must have clicked the wrong button when merging in something else. I’m not home right now but I’ll fix it later. 
> 
> > On Feb 4, 2021, at 6:59 PM, Kurt Sanders <notifications@github.com> wrote:
> > 
> > ﻿
> > I'm confused, did you close this pull request because you are not interested in these new features? Just curious?
> > 
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub, or unsubscribe.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/dcmeglio/hubitat-thinq/pull/13#issuecomment-773688418>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AC4SJOWVE3Q3RM4UDAOLAEDS5MZ2XANCNFSM4XDUDXNQ>.
>

KurtSanders commented:
Hang tight, I have rebased my master branch with yours, and added my enhancements this morning.  I will need to test with the new code to verify that everything works as expected!

If they do work as expected, I will send another pull request for your consideration!


¸¸.•*¨*•*´¨)
¸.•´¸.•*´¨) ¸.•*¨)
(¸.•´ (¸.•` ¤ 

Kurt Sanders
2485 Ridge Road
Xenia, Ohio 45385
+1937-723-0010

> On Feb 4, 2021, at 7:13 PM, dcmeglio <notifications@github.com> wrote:
> 
> 
> 
> 
> No. I must have clicked the wrong button when merging in something else. I’m not home right now but I’ll fix it later. 
> 
> > On Feb 4, 2021, at 6:59 PM, Kurt Sanders <notifications@github.com> wrote:
> > 
> > ﻿
> > I'm confused, did you close this pull request because you are not interested in these new features? Just curious?
> > 
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub, or unsubscribe.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/dcmeglio/hubitat-thinq/pull/13#issuecomment-773688418>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AC4SJOWVE3Q3RM4UDAOLAEDS5MZ2XANCNFSM4XDUDXNQ>.
>
